intl/msg_hash_us.h

```diff
@@ -11974,7 +11954,7 @@ MSG_HASH(
    MENU_ENUM_LABEL_VALUE_SUSTAINED_PERFORMANCE_MODE,
    "Sustained Performance Mode"
    )
-MSG_HASH(
+MSG_HASH( /* FIXME Unused? */
```

Comment:
This string does appear, mostly in consoled-based versions of RetroArch.
![PAL60](https://user-images.githubusercontent.com/5641681/111034434-f71d3c00-8415-11eb-8e24-de96e76e6a67.png)

Reply:
Really? But I can't find the hash (MENU_ENUM_LABEL_VALUE_PAL60_ENABLE) being used anywhere in the code. Weird.
Well, shows what I know, I guess.

Reply:
This screenshot was made a few months ago, so I do not know if this option was actually deleted. You should get a PS3 daily and see if it still has this.

Reply:
Uh, do I need a PS3 to check that? Or how else can I do it?

Reply:
Not really. RPCS3 can emulate the UI.

Reply:
Ah yes, I forgot that was an option. Anyway, after finally getting it to run I can confirm that the option in question is no longer there:
![Screenshot from 2021-03-13 23-28-16](https://user-images.githubusercontent.com/49207524/111051565-4f732e80-8454-11eb-9322-8f0337cca36a.png)